youtube_dl/extractor/sbs.py

```diff
@@ -10,15 +12,29 @@
 
 class SBSIE(InfoExtractor):
     IE_DESC = 'sbs.com.au'
-    _VALID_URL = r'https?://(?:www\.)?sbs\.com\.au/(?:ondemand|news)/video/(?:single/)?(?P<id>[0-9]+)'
+    _VALID_URL = r'https?://(?:www\.)?sbs\.com\.au/(?:ondemand|news)/(?:video/)?(?:single/)?([0-9]+|[0-9a-z-]+)'
```

Comment:
Must be a separate extractor delegating to SBS.

Reply:
Thanks for your review, @dstftw. I agree, this does feel like a separate extractor. Addressed in 744df7ab325c6ee28c5a877002adfba91f0e7a36 . Let me know if you need anything further.